produce/README.md

```diff
@@ -35,7 +35,9 @@ produce
 
 ###### Create new iOS apps on iTunes Connect and Dev Portal using your command line
 
-Get in contact with the developer on Twitter: [@FastlaneTools](https://twitter.com/FastlaneTools)
+Produce creates new iOS apps on both the Dev Portal and iTunes Connect. Produce sets the minimum required information for a new app and is designed to be minimalistic in nature.
+
+Get in contact with the developers on Twitter: [@FastlaneTools](https://twitter.com/FastlaneTools)
```

Comment:
"`produce` creates new iOS apps on both the Apple Developer Portal and iTunes Connect with the bare minimum information."